STANDARD.md

```diff
@@ -233,9 +233,10 @@ Required Attributes for the `basePath`
 --------------------------------------
```

Comment:
The title in line 232 still reads "Required" :)
Do we want to make this optional or recommended?